### ISSUE TITLE
question re tqdm and progress meter

### ISSUE THREAD

**Issue by spoo333** [DarwinAwardWinner/rganalysis#13]
Excuse the potentially ridiculous question. I've got tqdm installed to be able to watch the analysis progress, but something looks different these days (it's been like this for a while). Could this have to do with changes in the underlying structures, ie not an rganalysis issue as such? I don't know how to describe it properly, but I seem to remember that the progress meter was more detailed in the past.

**Comment by spoo333** [comment]
To be more precise. Now it says, e.g, "Analyzing: 1it [02:07, 127.20s/it]" when it starts analysing an album, but doesn't show progress for individual tracks. This is purely cosmetic of course and doesn't diminish the function of this excellent script whatsoever :)

**Comment by DarwinAwardWinner** [comment]
I haven't made any recent changes to rganalysis, so I assume it's a change in one of the dependencies. I'll try to investigate when I have the time, but it might not be fixable from rganalysis.

**Comment by spoo333** [comment]
Yes I'm certain it's a change in the dependencies because it changed from one day to the next without any revisions in the main script. It's not strictly necessary but would be fun to have back. Light show kind of thing :) thanks for looking into it.
